src/main.ts

```diff
@@ -76,9 +77,22 @@ async function run(): Promise<void> {
       )
     }
 
-    const regex = new RegExp(`/${filter}\\b`, 'g')
-    // Get the changed files from the response payload.
-    const files = response.data.files.filter(file => file.filename.match(regex))
+    const files = response.data.files.filter(file => {
+      let match = false
+      for (const item of globFilter) {
+        const pattern = item
+        core.debug(`Test ${file.filename} against ${pattern}`)
+        core.debug(`current match value: ${match}`)
+        if (pattern.includes('!')) {
```

Comment:
should we be more explicit here and check that it starts by`!` instead of including it?